gateway/server.go

```diff
@@ -993,11 +992,14 @@ func Start() {
 		mainLog.Warn("The control_api_port should be changed for production")
 	}
 
-	start()
+	checkup.FileDescriptors()
```

Comment:
I would put everything in one “checkup.Run” public function. This way every time we add something wouldn’t require an update in here and all the checkups would be incapsulated inside “Run”.

Reply:
In general, agree. It will require modifying code only in one place if we want to add similar warnings in future.

Reply:
In general, agree. It will require modifying code only in one place if we want to add similar warnings in future.